auth4genai/snippets/get-started/prerequisites/call-others-api.jsx

```diff
@@ -115,6 +115,24 @@ export const Prerequisites = ({
     );
   }
   // Always include these final steps
+
+  steps.push(
+    <Step key="my-account-api" title="Configure My Account API">
+      In the Auth0 Dashboard, configure the <a href="https://auth0.com/docs/manage-users/my-account-api" target="_blank">My Account API</a> to enable <a href="/intro/account-linking">account linking</a>:
```

Comment:
should we mention _why_ they need to do this? It's specifically needed to enable the Connect Account flow (the flow is started by calling an endpoint of the My Account API). Is that too much details to include in this section?